lib/SlugifierInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Api\Slugifier;
+
+/**
+ * Slugifier interface
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+interface SlugifierInterface
+{
+    /**
+     * Return a slugified (or urlized) representation of a given string
+     *
+     * @param string
+     *
+     * @return string
+     */
+    public function slugify($string);
```

Comment:
Maybe we should expand this documentation to be more detailed about what is expected.